Add `enable_automatic_upgrade` option to `virtual_machine_scale_set_extension` (#13394)
hashicorp/terraform-provider-azurerm: website/docs/r/virtual_machine_scale_set_extension.html.markdown

* `auto_upgrade_minor_version` - (Optional) Should the latest version of the Extension be used at Deployment Time, if one is available? This won't auto-update the extension on existing installation. Defaults to `true`.

> * `automatic_upgrade_enabledd` - (Optional) When Automatic Extension Upgrade is enabled on a VM or scale set, the extension is upgraded automatically whenever the extension publisher releases a new version for that extension. Defaults to `false`.

Reviewer: can we update this terminology to match the other resources:

```suggestion
* `automatic_upgrade_enabled` - (Optional) Should the Extension be automatically updated whenever the Publisher releases a new version of this VM Extension? Defaults to `false`.
```
